modules/openapi-generator/src/main/resources/rust/reqwest/api.mustache

```diff
@@ -56,7 +56,7 @@ impl {{classname}} for {{classname}}Client {
         let query_string = {
             let mut query = ::url::form_urlencoded::Serializer::new(String::new());
 {{#queryParams}}
-            query.append_pair("{{baseName}}", &{{paramName}}{{#isListContainer}}.join(","){{/isListContainer}}.to_string());
+            query.append_pair("{{baseName}}", &{{paramName}}{{#isListContainer}}.iter().map(|p| p.to_string()).collect::<Vec<String>>().join(","){{/isListContainer}}.to_string());
```

Comment:
Nice.

For bonus points, you could use `into_iter` instead of `iter`. As I understand it, `iter` produces references to `p`, and `to_string` will therefore clone `p`. `into_iter` would give you an owned `p`, and thus `to_string` would not clone..